index.js

```diff
@@ -82,3 +85,20 @@ function search() {
     })
   })
 }
+
+function addContact() {
+  $("#add-contact").on("click", function() {
+    $(".new-contact").fadeIn()
+  })
+}
+
+function create() {
```

Comment:
This is cool. I like the `fadeIn` and `fadeOut` parts. Again, I would separate out DOM manipulation / click events from pure functions.